tests/dummy/public/test-forms/dynamic-behaviour-v2/form.ttl

```diff
@@ -0,0 +1,144 @@
+    sh:name "More comment 1 star rating" ;
+    sh:order 220 ;
+    sh:path ext:ratingComment ;
+    form:options """{}""" ;
+    form:displayType displayTypes:textArea ;
+
+    form:partOf sections:main ;
+
+    ##########################################################
+    # Condition to trigger 1 star field
+    ##########################################################
+    form:rendersWhen
+      [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path ext:rating ;
+        form:conceptScheme <http://redpencil.data.gift/concept-schemes/21d1250c-6627-4111-934f-f2f7bd8c078a> ;
+        form:customValue <http://redpencil.data.gift/concepts/1770b75a-95d7-4087-93d6-2c9322b7dfee>
+      ] .
+
+##########################################################
+# More comment 2 stars rating
+##########################################################
+fields:comment-two-stars a form:Field ;
+    mu:uuid "43dd5003-c9a4-48c6-b650-b90f2bce8c7d";
+    sh:name "More comment 2 stars rating" ;
+    sh:order 220 ;
+    sh:path ext:ratingComment ;
+    form:options """{}""" ;
+    form:displayType displayTypes:textArea   ;
+
+    form:partOf sections:main ;
+    ##########################################################
+    # Condition to trigger 2 star field
+    ##########################################################
+    form:rendersWhen
+      [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path ext:rating ;
+        form:conceptScheme <http://redpencil.data.gift/concept-schemes/21d1250c-6627-4111-934f-f2f7bd8c078a> ;
+        form:customValue <http://redpencil.data.gift/concepts/0d5b3b23-c055-4d56-83d0-b65d28050315>
+      ] .
+
+
+##########################################################
+#  property-group: subset of star rating
+##########################################################
+fields:subset-property-group a form:Section;
+    sh:description "parent property-group, used to group fields and property-groups together";
+    sh:name "Select a value, if it matches subset of ratings it will trigger a text area" ;
+    sh:order 2 .
+
+##########################################################
+# Dropdown
+##########################################################
+fields:star-rating-subset a form:Field ;
+    sh:name "Triggers textarea if 3 or 4 stars is given " ;
+    sh:order 100 ;
+    sh:path ext:star-rating-subset-property ;
+    form:options """{"conceptScheme":"http://redpencil.data.gift/concept-schemes/21d1250c-6627-4111-934f-f2f7bd8c078a","searchEnabled":false}""" ;
+    form:displayType displayTypes:conceptSchemeSelector ;
+
+    form:partOf fields:subset-property-group .
+
+
+##########################################################
+# More comment subset stars rating
+##########################################################
+fields:more-comment-subset a form:Field ;
+    sh:name "More comments about star ratings 3,4" ;
+    sh:order 220 ;
+    sh:path ext:ratingComment ;
```

Comment:
I would use a different path for this one. At the moment it overrides the value of the first comment fields which is a bit confusing. The issue is already there in the original form, but I noticed it while testing this PR.

Reply:
had the same confusing experience while writing this but wanted to be as close as possible to the old example. Now that i see that I'm not alone in finding this annoying i will just change it in both examples